memgpt/local_llm/webui_settings.py

```diff
@@ -0,0 +1,54 @@
+DETERMINISTIC = {

```

Comment:
Should we put these inside a `/webui` folder? Something like:

```
/local_llm/webui/server_request.py
/local_llm/webui/settings.py
```

Reply:
Can we have some documentation in the README on what this file is for?